### PR TITLE
Praca domowa nr 3 🍺🍺🍺

### PR DESCRIPTION
**Pull request by Winkielek**

### CODE REVIEW

**Comment by ptomaszewska** [comment]
Poprawnie wykonana praca domowa.
- brakuje referencji dot. ablation study oraz źródła kodu modelu (czy jest autorski?)
- warto zadbać aby na wykresach nie było ułamkowych epok
- overfitting diagnozujemy na podstawie wykresu loss
- architektury sieci neuronowych można przedstawić jako "bloczki" - jest gotowa funkcja w kerasie
- prośba o zczytanie pracy domowej przed wysłaniem - jest dużo literówek co świadczyłoby, że praca została wykonana "na kolanie"
